packages/hardware-ledger/test/testData.ts

```diff
@@ -3,18 +3,25 @@ import { AddressType, KeyRole } from '@cardano-sdk/key-management';
 import { Base64Blob, HexBlob } from '@cardano-sdk/util';
 import { Cardano, Serialization } from '@cardano-sdk/core';
 import { LedgerTxTransformerContext } from '../src';
-export const rewardAccount = Cardano.RewardAccount('stake1u89sasnfyjtmgk8ydqfv3fdl52f36x3djedfnzfc9rkgzrcss5vgr');
+export const rewardAccount = Cardano.RewardAccount('stake_test1up7pvfq8zn4quy45r2g572290p9vf99mr9tn7r9xrgy2l2qdsf58d');
 export const stakeKeyHash = Cardano.RewardAccount.toHash(rewardAccount);
 export const stakeCredential = {
-  hash: stakeKeyHash as unknown as Crypto.Hash28ByteBase16,
+  hash: Crypto.Hash28ByteBase16.fromEd25519KeyHashHex(stakeKeyHash),
+  type: Cardano.CredentialType.KeyHash
+};
+
+export const dRepCredential = {
+  hash: Crypto.Hash28ByteBase16('b276b4f7a706a81364de606d890343a76af570268d4bbfee2fc8fcab'),
   type: Cardano.CredentialType.KeyHash
 };
 export const paymentAddress = Cardano.PaymentAddress(
   'addr1qxdtr6wjx3kr7jlrvrfzhrh8w44qx9krcxhvu3e79zr7497tpmpxjfyhk3vwg6qjezjmlg5nr5dzm9j6nxyns28vsy8stu5lh6'
 );
 export const paymentHash = Crypto.Ed25519KeyHashHex('9ab1e9d2346c3f4be360d22b8ee7756a0316c3c1aece473e2887ea97');
-export const ownerRewardAccount = Cardano.RewardAccount('stake1u89sasnfyjtmgk8ydqfv3fdl52f36x3djedfnzfc9rkgzrcss5vgr');
-export const poolId = Cardano.PoolId('pool1ev8vy6fyj7693ergzty2t0azjvw35tvkt2vcjwpgajqs7z6u2vn');
+export const ownerRewardAccount = Cardano.RewardAccount(
+  'stake_test1up7pvfq8zn4quy45r2g572290p9vf99mr9tn7r9xrgy2l2qdsf58d'
+);
```

Comment:
Do you need this? It is the same as https://github.com/input-output-hk/cardano-js-sdk/pull/1211/files#diff-86b266e5ed64ea085b12579f6e18bb9659eb54d9b73fd39b34608ca4af176534R6

Reply:
done here [0f6bc6de2d8009ed43f47f4cf3a7b1f222160349](https://github.com/input-output-hk/cardano-js-sdk/pull/1211/commits/0f6bc6de2d8009ed43f47f4cf3a7b1f222160349)